src/urls.py

```diff
@@ -0,0 +1,27 @@
+from django.urls import path
+
+from . import tasks
+
+urlpatterns = [
+    path('', tasks.index, name='index'),
+
+
+    path('task1', tasks.Task1.as_view(), name='task1'),
```

Comment:
what is the need for exposing two different URLs? We can have one url and then call the function inside and return the result, isn't it?

Reply:
which 2 urls?

Reply:
path('task1', tasks.Task1.as_view(), name='task1'),
path('execute_task1', tasks.execute_task1, name='ExecuteTask1')

I think I got it 1st one is for displaying the form when Task1 is selected and the second one is for POST requests when the task is executed.